DEVELOPMENT.md

```diff
@@ -0,0 +1,50 @@
+# Notes for developers
+
+## Building
+
+You need Go version 1.14 to build this project.
+
+```shell
+$ go build ./cmd/services
+```
+
+## Testing
+
+Linting should be done first (this is done on Travis, and what's good locally should be good there too)
+
+Grab the linter if you haven't already: 
+
+```shell
+GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.26.0
+```
```

Comment:
If we're using Go 1.13+ do we need `GO111MODULE=on` given that the default `auto` does the right thing in almost all cases?

Reply:
Trying this in a clean environment:
```
% docker run -it golang bash
root@2dd3d15955e1:/go# go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.26.0
go: cannot use path@version syntax in GOPATH mode
```
The command succeeds if prefixed with `GO111MODULE=on` so I'm going to say yes it's still needed.

Reply:
Ahhh...indeed, tho', maybe it should link to the installation instructions for golangci-lint ?